lib/agenda.js

```diff
@@ -0,0 +1,41 @@
+const config = require('config');
```

Comment:
I don't particularly like the fact that we call this `agenda`. Maybe we can use a more generic term that doesn't reference the library that we're using.

Reply:
Named it "job-queue" but open to suggestions.